lib/partition_setup.pm

```diff
@@ -406,7 +406,7 @@ sub take_first_disk_storage_ng {
     assert_screen [qw(existing-partitions partition-scheme)];
     # If drive(s) is/are not formatted, we have select hard disks page
     if (match_has_tag 'existing-partitions') {
-        if (check_var('BACKEND', 'ipmi')) {
+        if (check_var('BACKEND', 'ipmi') && !chech_var('DESKTOP', 'textmode')) {
```

Comment:
is that similiar to slovak_var?

Reply:
@coolo I don't quite understand.

Reply:
Sorry, chech_var looks like localization to me - and my humour just awoke :)